packages/utilities/src/properties.ts

```diff
@@ -231,6 +231,13 @@ export const buttonProperties = htmlElementProperties.concat(['disabled']);
  */
 export const divProperties = htmlElementProperties.concat(['align', 'noWrap']);
 
+/**
+ * An array of LABEL tag properties and events.
+ *
+ * @public
+ */
+export const labelProperties = htmlElementProperties.concat(['for', 'form']);
```

Comment:
Would we never need 'align' or 'noWrap' here, as added in `divProperties` above?

Reply:
Just curious given that Label used `divProperties` before

Reply:
I'm gonna see if I can test out not mucking with this and just use "htmlFor" - although divProps seems wrong